Common/Transforms/itkAdvancedBSplineDeformableTransform.hxx

```diff
@@ -192,25 +192,11 @@ auto
 AdvancedBSplineDeformableTransform<TScalarType, NDimensions, VSplineOrder>::TransformPoint(
   const InputPointType & point) const -> OutputPointType
 {
-  /** Allocate memory on the stack: */
-  const unsigned long                         numberOfWeights = WeightsFunctionType::NumberOfWeights;
-  typename ParameterIndexArrayType::ValueType indicesArray[numberOfWeights];
-  WeightsType                                 weights;
-  ParameterIndexArrayType                     indices(indicesArray, numberOfWeights, false);
-
-  OutputPointType outputPoint;
-
-  InputPointType transformedPoint = point;
-
   /** Check if the coefficient image has been set. */
   if (!this->m_CoefficientImages[0])
   {
     itkWarningMacro(<< "B-spline coefficients have not been set");
-    for (unsigned int j = 0; j < SpaceDimension; ++j)
-    {
-      outputPoint[j] = transformedPoint[j];
-    }
-    return outputPoint;
+    return point;
```

Comment:
should this not be transformedPoint?

Reply:
Thanks for reviewing, @mstaring but I think it's OK to just do `return point` here. The original code did: https://github.com/SuperElastix/elastix/blob/59ceb4fc01cdce8d457d105157ef47f72691b1b8/Common/Transforms/itkAdvancedBSplineDeformableTransform.hxx#L203-L214

So it copied `point` to `transformedPoint`, and then it copied the coordinates of `transformedPoint` to `outputPoint`. Then it returned `outputPoint`. In this particular case (if the coefficient image has _not_ been set), I think it's ok to return the input point directly. Right?